Commands/Public/wiki.js

```diff
@@ -0,0 +1,59 @@
+const wiki = require("wikijs").default();
+
+module.exports = async ({ Constants: { Colors } }, documents, msg, commandData) => {
+	await msg.send({
+		embed: {
+			color: Colors.INFO,
+			title: `Searching Wikipedia just for you ⌛`,
+			description: `Please stand by...`,
+		},
+	});
+	let result;
+	if (!msg.suffix) {
+		const random = await wiki.random(1);
+		result = await wiki.page(random[0]);
+	} else {
+		const search = await wiki.search(msg.suffix, 1);
+		if (!search.results.length) {
+			return msg.send({
+				embed: {
+					color: Colors.SOFT_ERR,
+					title: "What was that again? 📚🤓",
+					description: "The almighty Wikipedia could not find a result for your search query.",
```

Comment:
Almighty is too cheesy.
Sorry, but "A result could not be found for your search query!"

Reply:
Boo.